lib/_emerge/Package.py

```diff
@@ -860,9 +860,10 @@ def with_use(self, use):
         return pkg
 
 
-_all_metadata_keys = set(x for x in portage.auxdbkeys)
-_all_metadata_keys.update(Package.metadata_keys)
-_all_metadata_keys = frozenset(_all_metadata_keys)
+_all_metadata_keys_temp = set(x for x in portage.auxdbkeys)
+_all_metadata_keys_temp.update(Package.metadata_keys)
+_all_metadata_keys = frozenset(_all_metadata_keys_temp)
+del _all_metadata_keys_temp
```

Comment:
This change is pretty ugly. Can you come up with a way to avoid using a temporary variable?

Reply:
Yeah, I think I can.

Reply:
I've removed the temp var.